README.md

```diff
@@ -80,28 +75,29 @@ To enable digest pinning, pass the following arguments:
 
 ```bash
   --param pin_digests=true \
+  --param git_repo_url=<github_repo_ssh_url> \
```

Comment:
That's intentional to make it easier to develop/test when digest pinning is not desired. By recommending HTTPS by default for developers, there's no need to configure the SSH secret.

`git_repo_url` isn't needed in the `commit-pinned-digest` task because the source should have already been checked out to the pipeline workspace using SSH. The optional `ssh-workspace` is required in this case.